src/Core/Components/InputFile/FluentInputFileEventArgs.cs

```diff
@@ -42,6 +42,11 @@ public class FluentInputFileEventArgs : EventArgs
     /// </summary>
     public string ContentType { get; internal set; } = string.Empty;
 
+    /// <summary>
+    /// Gets the last modified date of the current file in an upload process.
+    /// </summary>
+    public DateTimeOffset LastModified { get; internal set; } = default!;
```

Comment:
Can you add a Unit Test to validate this new feature?
You will find some examples in the `FluentInputFileTests.razor` file

Reply:
Yes, I have added unit tests now